srcpkgs/betterlockscreen/template

```diff
@@ -1,16 +1,18 @@
 # Template file for 'betterlockscreen'
 pkgname=betterlockscreen
-version=4.0.4
+version=4.4.0
 revision=1
 depends="ImageMagick bash bc feh i3lock-color xdpyinfo xrandr xrdb xset"
 short_desc="Sweet looking lockscreen for linux system"
 maintainer="Aleksander Zien <Aleks@nderZien.com>"
 license="MIT"
 homepage="https://github.com/pavanjadhaw/betterlockscreen"
 distfiles="https://github.com/pavanjadhaw/betterlockscreen/archive/refs/tags/v${version}.tar.gz"
-checksum=608b56989eea0b45c0f88bdc9e3c7742430f809a1bcd60d558eb6af99aeecc91
+checksum=fdcf58725f0ab56e98ce9b83bd1dad6e1f08c7f9f230c3abaf4191f10ebd5dfd
 
 do_install() {
+	vsed -i betterlockscreen -e 's/suspend_command="systemctl suspend"/suspend_command="loginctl suspend"/g'
+	vsed -i examples/betterlockscreenrc -e 's/suspend_command="systemctl suspend"/suspend_command="loginctl suspend"/g'
```

Comment:
better to fix the patch than vsed, it's not a complex patch

Reply:
Ah ah ah I thought I was smart to replace the patch by vsed 😉 
Thanks for the feedback. Should be fixed now.